katrain/gui/badukpan.py

```diff
@@ -752,6 +758,12 @@ def set_animating_pv(self, pv, node):
             self.animating_pv = (pv, node, time.time(), self.last_mouse_pos)
         self.redraw_hover_contents_trigger()
 
+    def increase_animate_pv_index(self, delta=1):
+        if self.animating_pv_index + delta < 0:
```

Comment:
maybe

if index is None:
  index = current one based on delay (ignore delta)
else:
  index = max(0, index+delta)